core/src/sigverifier/bls_sigverifier/stats.rs

```diff
@@ -41,6 +42,7 @@ impl BLSSigVerifierStats {
             preprocess_count: AtomicU64::new(0),
             preprocess_elapsed_us: AtomicU64::new(0),
             votes_batch_count: AtomicU64::new(0),
+            votes_batch_distinct_messages_count: AtomicU64::new(0),
```

Comment:
You need to add this to datapoint_info! below as well.

Reply:
Okay fixed!